beastify/manifest.json

```diff
@@ -1,5 +1,6 @@
 {
 
+  "description": "Adds a browser action icon to toolbar to select a beast from popup menu to replace document body content with an image of the beast in active tab.",
```

Comment:
This sentence is pretty long. Could we say: "Adds a browser action icon to the toolbar. Click the button to choose a beast. The active tab's body content is then replaced with a picture of the chosen beast."